src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -301,9 +689,15 @@ public void binaryContentNoTypeDefault(final String uri) {
     public void binaryContentNoTypeUnsupported(final String uri) {
         final TestInfo info = setupTest("3.9-F-4", "binaryContentNoTypeUnsupported",
                 "Fedora servers must use the value of the type attribute in the external content link as the media " +
-                        "type of the external content, if provided. Servers may reject the request with a 4xx range " +
-                        "status code.",
+                        "type of the external content, if provided. If there is no type attribute: " +
+                        "Servers may reject the request with a 4xx range status code.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUriWithNoType, "proxy")),
+                new Header(SLUG, info.getId()));
```

Comment:
Please avoid using the `Slug` header where possible as support for it is not required AFAIK.